src/Mod/Mesh/App/Exporter.cpp

```diff
@@ -38,17 +39,49 @@
 #include "Base/Stream.h"
 #include "Base/Tools.h"
 
-#include "App/Part.h"
+#include "App/Application.h"
+#include "App/ComplexGeoData.h"
+#include "App/ComplexGeoDataPy.h"
+#include "App/DocumentObject.h"
 
 #include <zipios++/zipoutputstream.h>
 
 using namespace Mesh;
 using namespace MeshCore;
 
-Exporter::Exporter() :
-    meshFeatId( Base::Type::fromName("Mesh::Feature") ),
-    appPartId( Base::Type::fromName("Part::Feature") ),
-    groupExtensionId( App::GroupExtension::getExtensionClassTypeId() )
+static std::vector<std::string>
+expandSubObjects(const App::DocumentObject *obj,
```

Comment:
```suggestion
expandSubObjectNames(const App::DocumentObject *obj,
```
to convey with greater clarity what the function does.